src/wasm/nxt_wasm.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#ifndef _NXT_WASM_H_INCLUDED_
+#define _NXT_WASM_H_INCLUDED_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <nxt_unit.h>
+
+#include <wasm.h>
+#if defined(NXT_HAVE_WASM_WASMTIME)
+#include <wasmtime.h>
+#endif
+
+#define NXT_WASM_PAGE_SIZE          (64 * 1024)
+
+#define NXT_WASM_MEM_PAGES          512UL       /* 32 MiB */
+
+#define NXT_WASM_MEM_SIZE           (NXT_WASM_PAGE_SIZE * NXT_WASM_MEM_PAGES)
+
+#if defined(NXT_HAVE_WASM_WASMTIME)
+typedef wasmtime_func_t  nxt_wasm_func_t;
+#endif
+
+typedef struct {
+    uint32_t  name_offs;
+    uint32_t  name_len;
+    uint32_t  value_offs;
```

Comment:
Does offs mean plural?  I think I prefer `_off` for singular.

Reply:
offs = offset

Reply:
I think off reads better.  Maybe it's because of `off_t`...

Reply:
Hmm, think I've figured out how I lost all my comments.

Seems if you comment **AND** click on 'Resolve conversation' it doesn't record the comment.

Anyway changed to _off